CONTRIBUTING.md

```diff
@@ -84,8 +84,7 @@ In order to make reviews simpler and have contributions merged faster in the cod
 [5]: https://github.com/Subsurface-divelog/subsurface/issues
 [6]: https://github.com/Subsurface-divelog/subsurface/pulls
 [7]: https://www.kernel.org/pub/software/scm/git/docs/user-manual.html
-[8]: https://subsurface-divelog.org/building/
-[9]: https://gerrit-review.googlesource.com/Documentation/user-signedoffby.html
-[10]: https://subsurface-divelog.org/wp-content/uploads/2011/10/Screenshot-gitk-subsurface-1.png "Example with gitk"
-[11]: https://github.com/Subsurface-divelog/subsurface/blob/master/CodingStyle
-[12]: https://github.com/Subsurface-divelog/subsurface/blob/master/CONVENTIONS.md
+[8]: https://gerrit-review.googlesource.com/Documentation/user-signedoffby.html
+[9]: https://subsurface-divelog.org/wp-content/uploads/2011/10/Screenshot-gitk-subsurface-1.png "Example with gitk"
+[10]: https://github.com/Subsurface-divelog/subsurface/blob/master/CodingStyle
+[11]: https://github.com/Subsurface-divelog/subsurface/blob/master/CONVENTIONS.md
```

Comment:
this changes the link references, without changing the references to those links (i.e., where we reference `[9]` in the file we should reference `[8]` now, correct?

Reply:
Damnit, you're right. I'll fix that now.

Reply:
Ok, that should now be fixed.

Not making a great first impression really am i, a bit like an overenthusiastic puppy!

Reply:
You are making an outstanding impression

#1 thing I value in a contributor? Positive response to feedback on a pull request. And then, ideally, updates based on that feedback...